apps/ui/src/composables/useWhiteLabel.ts

```diff
@@ -0,0 +1,73 @@
+import { getNetwork } from '@/networks';
+import { useSpacesStore } from '@/stores/spaces';
+import { Space } from '@/types';
+
+const NETWORK = 's';
```

Comment:
Should we use this instead?
https://github.com/snapshot-labs/sx-monorepo/blob/4bfba1c11252bdbab241677ca7734ec7f0828dbb/apps/ui/src/networks/index.ts#L44-L45